benchexec/tools/svf.py

```diff
@@ -0,0 +1,62 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from benchexec.tools.sv_benchmarks_util import get_data_model_from_task, ILP32, LP64
+import benchexec.tools.template
+import benchexec.result as result
+
+
+class Tool(benchexec.tools.template.BaseTool2):
+    """
+    Tool info for SVF: a framework for static value-flow analysis.
+    Specifically this tool is a wrapper around SVF to make it work with SV-COMP.
```

Comment:
I am wondering what the relation between SVF and svf-svc-svcomp is. Is the latter an official project by the SVF team? Or is it a third-party wrapper that implements something on top of SVF?

Right now this tool-info module claims to be for SVF, but it looks as if one cannot actually execute SVF with it, only svf-svc-svcomp. I guess then the tool-info module's name should probably reflect that. If later someone comes and wants to benchmark proper SVF with BenchExec, they should be able to submit a tool-info module named `svf.py`.

Reply:
The  submission is built on top of SVF and use SVF as a library. I would say the naming svf-svc is more appropriate name as the submission is tailored for this competition.

Reply:
Thanks Yulei. I've updated the name to reflect this.

Reply:
Thanks. Note that you can also name your tool-info module `svf-svc` if you prefer, even though the dash is normally not allowed in Python module names.